src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.test.tsx

```diff
@@ -171,23 +183,34 @@ describe('FixPhoneNumbers-Home', () => {
     });
 
     userEvent.click(getByTestId('confirmButton-testid'));
+
     await waitFor(() => {
-      expect(mockEnqueue).toHaveBeenCalledWith('Phone numbers updated!', {
-        variant: 'success',
-      });
+      expect(mockEnqueue).toHaveBeenCalledWith(
+        'Successfully updated phone numbers for Test Contact',
+        {
+          variant: 'success',
+        },
+      );
     });
   });
   it('should bulk confirm all phone numbers', async () => {
     const { getByTestId, queryByTestId, getByText } = render(<Components />);
     await waitFor(() => {
       expect(queryByTestId('loading')).not.toBeInTheDocument();
-      expect(getByTestId('starOutlineIcon-testid-1')).toBeInTheDocument();
+      expect(getByTestId('starOutlineIcon-testid-id2')).toBeInTheDocument();
+    });
+
+    userEvent.click(getByTestId(`starOutlineIcon-testid-id2`));
+
+    fireEvent.change(getByTestId('source-select'), {
+      target: { value: 'MPDX' },
     });
```

Comment:
I think this could use `userEvent.click` also. You probably have to click on `source-select` and then click on `getByRole('option', { name: "Item name" })`.

Reply:
Is the selectbox disabled?

Reply:
Oh, the input has `aria-hidden="true"` on it. I think what you want to do is put the select and the "Default Primary Source" label in a `<FormControl>` like [here](https://mui.com/material-ui/react-select/#basic-select). That will be better for accessibility too because "Default Primary Source" will be associated with the select box. Then in the test you can find it with `getByRole('combobox', { name: 'Default Primary Source '})`.